### PR TITLE
Shyrma mmul

### PR DESCRIPTION
**Pull request by shyrma**
mmul helper has been improved

### CODE REVIEW

File: libnd4j/include/cblas.h [hunk 1]
```diff
@@ -21,7 +21,7 @@
 #ifndef NATIVEOPERATIONS_CBLAS_H
 #define NATIVEOPERATIONS_CBLAS_H
 
-#include "config.h"
+// #include "config.h"
```
Comment (saudet) [inline libnd4j/include/cblas.h]: This will break the build, please don't do that.

File: libnd4j/include/graph/impl/GraphUtils.cpp [hunk 2]
```diff
@@ -18,7 +18,7 @@
 // Created by GS <sgazeos@gmail.com> 3/7/2018
 //
 
-#include "config.h"
+// #include "config.h"
```
Comment (saudet) [inline libnd4j/include/graph/impl/GraphUtils.cpp]: Here too, please don't comment those out.

File: libnd4j/include/cblas.h (hunk 1 above)
Comment (raver119) [reply libnd4j/include/cblas.h]: Don't worry, it's temporary comment.

File: libnd4j/include/graph/impl/GraphUtils.cpp (hunk 2 above)
Comment (raver119) [reply libnd4j/include/graph/impl/GraphUtils.cpp]: Don't worry, it's temporary comment.

**Pull request merged**